app/controllers/application_controller.rb

```diff
@@ -74,4 +74,39 @@ def set_no_cache_header
   def touch_current_user
     current_user.touch
   end
+
+  # The following is from the now deprecated fastly-rails gem
```

Comment:
Thoughts on putting this in a module so all the fastly logic is contained in a single file rather than just at the bottom of the ApplicationController? Might be easier to find it that way in the codebase.